qprofiler2/test/org/qcmg/qprofiler2/OptionsTest.java

```diff
@@ -3,10 +3,10 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.fail;
-
 import org.junit.Test;
 import org.qcmg.qprofiler2.Options2;
 
+
```

Comment:
no changes in this class

Reply:
added option "--fullBamHeader", but tested in QProfiler2Test.java